platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/EcmsManageDocumentsInSpaceTestIT.java

```diff
@@ -1,102 +1,139 @@
+        homePagePlatform.goToMySpaces();
+        ELEMENT_SPACES_LIST.find(byText(space)).click();
+        spaceManagement.goToDocumentTab();
+        $(byId("MultiUploadInputFiles")).uploadFromClasspath("eXo-Platform.png");
+        ELEMENT_LIST_DOCUMENTS_IN_SPACE.find(byText("eXo-Platform.png")).waitUntil(Condition.visible, Configuration.timeout).click();
+        documentManagement.goToPermissions();
+        $(byId(ELEMENT_CHECKBOX_MODIFY_PERMISSION.replace("{user}", space))).parent().waitWhile(Condition.not(Condition.visible),
+                Configuration.timeout);
+        if ($(byId(ELEMENT_CHECKBOX_MODIFY_PERMISSION.replace("{user}", space))).is(Condition.checked))
+            $(byId(ELEMENT_CHECKBOX_MODIFY_PERMISSION.replace("{user}", space))).parent().click();
+        $(ELEMENT_CLOSE_BTN).click();
+        manageLogInOut.signIn(PLFData.DATA_USER2, password);
+        homePagePlatform.goToMySpaces();
+        ELEMENT_SPACES_LIST.find(byText(space)).click();
+        spaceManagement.goToDocumentTab();
+        ELEMENT_LIST_DOCUMENTS_IN_SPACE.find(byText("eXo-Platform.png"))
+                .waitUntil(Condition.visible, Configuration.timeout)
+                .contextClick();
+        ELEMENT_BUTTON_UPLOAD_NEW_VERSION.shouldNotBe(Condition.visible);
+        ELEMENT_BUTTON_CREATE_NEW_VERSION.shouldNotBe(Condition.visible);
+        ELEMENT_LIST_DOCUMENTS_IN_SPACE.find(byText("eXo-Platform.png")).waitUntil(Condition.visible, Configuration.timeout).click();
+        ELEMENT_BUTTON_UPLOAD_NEW_VERSION.shouldNotBe(Condition.visible);
+        ELEMENT_BUTTON_CREATE_NEW_VERSION.shouldNotBe(Condition.visible);
+        manageLogInOut.signIn(PLFData.DATA_USER1, "gtngtn");
+        homePagePlatform.goToMySpaces();
+        spaceManagement.deleteSpace(space, false);
+
+    }
+
+    @Test
+    public void test02_CheckScriptNotExecutedWhenFilePathContainsScript() {
+        //8357
+        String space = "space" + getRandomNumber();
+        String folderTitle = "test<script>alert(2);</alert>";
+        String uploadedFile ="eXo-Platform.png";
+        info("Create a space");
+        homePagePlatform.goToMySpaces();
+        spaceManagement.addNewSpaceSimple(space, space);
+        homePagePlatform.goToDocuments();
+        siteExplorerHome.goToAddNewFolder();
+        info("Create Folder node");
+        spaceManagement.createFolder(folderTitle);
+        navigationToolbar.openFolderInDocuments(folderTitle);
+        $(byId("MultiUploadInputFiles")).uploadFromClasspath(uploadedFile);
+        navigationToolbar.checkFolderUploadedFile(uploadedFile);
+        homePagePlatform.goToMySpaces();
+        spaceManagement.searchSpace(space);
+        spaceManagement.goToSpace(space);
+        spaceManagement.checkUploadedFileNotExist();
+        homePagePlatform.goToDocuments();
+        spaceManagement.deleteFolder(folderTitle);
+        homePagePlatform.goToMySpaces();
+        spaceManagement.deleteSpace(space, false);
```

Comment:
Where do we check that the script is not executed ?

Reply:
Can you point me to the Jira issue please ?

Reply:
The Jira issue was
https://jira.exoplatform.org/browse/PLF-8357?filter=24603

Reply:
Thanks. The issue is that the script is executed, so we must check in this test (if possible) that the script is not executed anymore.

Reply:
It's clear ok.